README.md

```diff
@@ -105,17 +105,17 @@ In the examples, intentions are created in the Ansible playboook [00_create_inte
 | minio\_access\_key | Minio access key | string |
 | minio\_secret\_key | Minio secret key | string |
 
-## Credentials
+## Secrets & Credentials
 The minio access_key and secret_key is generated and put in `/secret/data/minio` inside Vault.
 
 To get the access_key and secret_key from Vault you can login to the [Vault-UI](http://localhost:8200/) with token `master` and reveal the access_key and secret_key in `/secret/minio`.
 Alternatively, you can ssh into the vagrant box with `vagrant ssh`, and use the vault binary to get the access_key and secret_key. See the following commands:
 ```sh
 # get access_key
-vault kv get -field='access_key' secret/data/minio
+vault kv get -field='access_key' secret/minio
```

Comment:
Is it the correct rollback?

Reply:
Yes. This will `secret/data/minio` will give error since the vault cli will look for path `sercret/data/data/minio` for some reason.

Reply:
Yes, in Vault you can only get them by doing secret/minio and not secret/data/minio. if you try and get with secret/data/minio you get nothing

Reply:
Looks like, you put it under `secret/minio´ -> https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/issue_66_creds_rendering/dev/ansible/01_generate_secrets_vault.yml#L7

Reply:
I mean here, you created a policy in the vault for `secrets/data/minio` https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/issue_66_creds_rendering/dev/vagrant/bootstrap/vault/post/01-create_vault_policy_to_read_secrets.yml#L4 

At the same, time you have put credentials under `secrets/minio` https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/issue_66_creds_rendering/dev/ansible/01_generate_secrets_vault.yml#L7 


What the point to have policy that you have defined in https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/issue_66_creds_rendering/dev/vagrant/bootstrap/vault/post/01-create_vault_policy_to_read_secrets.yml#L4 
 ?

Reply:
TBH, I have no idea. This is something that makes no sense, but it fails if the policy, path, and creds are set/put to `secrets/minio`... I found out (the hard way) in the Postgres module that the solution was to point the policy and path to `secrets/data/postgres` and put the credentials to `secrets/postgres`.

My best guess is that there might be a parser implemented in the Vault CLI which adds the secrets/**data**/postgres for us, but in the policy and path, it's not implemented that way so we have to specify it.